RELEASE_NOTES.md

```diff
@@ -1,6 +1,17 @@
 Release Notes
 =============
 
+v1.0.0-alpha.x
+--------------
+
+### New features
+
+- Adds the `Managed` management policy trait, that indicates that a
+  manager wishes to managed data matching the specified trait set.
```

Comment:
[fix][minor] Typo, `wishes to managed` -> `wishes to manage`

Reply:
Fixed [here](https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/18/files#diff-68d24fa81558aae3d8c59e2aa57a4fa719ea3b04d7fa14beff45f16f00858f50R10).